src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    public static Ball getRandomBall(){
+        Ball RandomBall = new Ball();
```

Comment:
Local variable names should start with a lowercase letter. So `RandomBall` should be `randomBall`.